artifact.go

```diff
@@ -338,10 +334,6 @@ func (a Artifact) inspectLayer(ctx context.Context, diffID string, disabled []an
 	opts := analyzer.AnalysisOptions{Offline: a.artifactOption.Offline}
 	result := analyzer.NewAnalysisResult()
 	limit := semaphore.NewWeighted(parallel)
```

Comment:
we have to make sure parrallel parameter will be 1 everywhere, otherwise there will be a lot OOM kills

Reply:
I've made it configurable via artifact options, so the caller gets to set whatever they want.